### PR TITLE
Create verified-builds.md

### PR DESCRIPTION
**Pull request by Woody4618**
### Problem
The Verify program project will soon move into the anzo tool suite. We don't have official docs on it yet. 

### Summary of Changes
Added docs for verify program and Security.txt

### CODE REVIEW

File: content/guides/advanced/verified-builds.md [hunk 1]
```diff
@@ -0,0 +1,356 @@
...
+security details are available to anyone looking to report potential issues.
+
+## Best practices
+
+- Use Links: For information likely to change (e.g., contact details), it's
+  recommended to link to a web page rather than hard-coding them into the
+  contract. This avoids the need for frequent program upgrades.
+
+- Verification: Before deploying, verify the format and content using the
+  query-security-txt tool, which can validate both onchain programs and local
```
Comment (nickfrosty) [inline content/guides/advanced/verified-builds.md:341]:
```suggestion
  `query-security-txt` tool, which can validate both onchain programs and local
```

File: content/guides/advanced/verified-builds.md [hunk 2]
```diff
@@ -0,0 +1,356 @@
+---
+date: 2024-09-26T00:00:00Z
+difficulty: intermediate
```
Comment (nickfrosty) [inline content/guides/advanced/verified-builds.md:3]: lets make this featured to show at the top of the guides page
```suggestion
difficulty: intermediate
featured: true
```

File: content/guides/advanced/verified-builds.md [hunk 3]
```diff
@@ -0,0 +1,356 @@
...
+The verification process is done by comparing the hash of the onchain program
+with the hash of the locally built program from the source code. You build your
+program in a controlled environment using the Solana Verify CLI and Docker. This
+ensures that the build process is deterministic and consistent across different
+systems. Once you have the executable, you can deploy it to the Solana network.
+During the build process a PDA of the
+[verify program](https://github.com/otter-sec/otter-verify) will be created.
+This PDA contains all the data necessary to verify the program.
+
+```rust
```
[the remark this answers is not in the document]
Comment (Woody4618) [reply content/guides/advanced/verified-builds.md:61]: It was supposed to shoe all the fields of the PDA. In included them in the text instead.

File: content/guides/advanced/verified-builds.md [hunk 4]
```diff
@@ -0,0 +1,356 @@
...
+For extra caution, you can install a version directly from a specific commit:
+
+```bash
+cargo install solana-verify --git https://github.com/Ellipsis-Labs/solana-verifiable-build --rev 13a1db2
+```
+
+3. Building Verifiable Programs
+
+To verify against a repository it needs to have a cargo.lock file in the root
+directory. You can for example use this as a preset:
```
[the remark this answers is not in the document]
Comment (Woody4618) [reply content/guides/advanced/verified-builds.md:150]: It needs to have a cargo.lock. But for that you need the cargo toml and then create the lock file. Updated.

File: content/guides/advanced/verified-builds.md (hunk 2 above)
Comment (Woody4618) [reply content/guides/advanced/verified-builds.md:3]: I would actually like to not put too much featuring on it until its actually ready. Wdyt? Still lots a work to do until its properly usable.

Comment (nickfrosty) [reply content/guides/advanced/verified-builds.md:3]: that's fair then, we can ignore making it featured.

do you think we should generally wait until we merge this then?

Comment (jacobcreech) [reply content/guides/advanced/verified-builds.md:3]: ship imo. Feature later. Get it indexed

File: content/guides/advanced/verified-builds.md [hunk 5]
```diff
@@ -0,0 +1,356 @@
+---
+date: 2024-09-26T00:00:00Z
+difficulty: intermediate
+title: "How to verify a program"
```
Comment (jacobcreech) [inline content/guides/advanced/verified-builds.md:4]:
```suggestion
title: "How to Verify a Program"
```

File: content/guides/advanced/verified-builds.md [hunk 6]
```diff
@@ -0,0 +1,356 @@
...
+network matches the source code in your repository. By doing this, developers
+and users can have confidence that the program running onchain corresponds
+exactly to the public codebase, promoting transparency and security.
+
+The verification process involves comparing the hash of the onchain program with
+the hash of the locally built program from the source code. This ensures no
+discrepancies between the two versions.
+
+> A verified build should not be considered more secure than a not verified
+> build. But it gives the user the ability to verify that the program was built
```
Comment (jacobcreech) [inline content/guides/advanced/verified-builds.md:38]:
```suggestion
> A verified build should not be considered more secure than an unverified build.
> A verified gives the user the ability to verify that the program was built
```

File: content/guides/advanced/verified-builds.md [hunk 7]
```diff
@@ -0,0 +1,356 @@
...
+> from the provided source code and by that everyone can verify the program does
+> what it is supposed to do and easily contact the developer and open an issue
+> if something goes wrong.
```
Comment (jacobcreech) [inline content/guides/advanced/verified-builds.md:41]:
```suggestion
> While a verified build should not be considered more secure than an unverified build, the build enables developers to self verify the source code matches what is deployed onchain. Using the source code, a developer can then validate what the code executes when sending a transaction.
```

File: content/guides/advanced/verified-builds.md [hunk 8]
```diff
@@ -0,0 +1,356 @@
...
+- Transparency: Allows other users and developers to validate that the onchain
+  program is trustworthy by comparing it with the public codebase.
+
+- Trust: Increase user confidence, as verified builds demonstrate that your
+  program's onchain behavior is aligned with your public code. When building
+  verifiable programs, you minimize risks associated with running unauthorized
+  or malicious code. It also ensures you comply with best practices and give
+  security researchers an easy way to contact you.
+
+# How do I implement verified builds?
```
Comment (jacobcreech) [inline content/guides/advanced/verified-builds.md:92]:
```suggestion
# How do I create verified builds?
```

**Review by jacobcreech — APPROVED** [review]

**Pull request merged**